internal/pkg/template/template.go

```diff
@@ -59,23 +84,36 @@ func (t *Template) Read(path string) (*Content, error) {
 
 // Parse parses the template under "/templates/{path}" with the specified data object and returns its content.
 func (t *Template) Parse(path string, data interface{}, options ...ParseOption) (*Content, error) {
-	content, err := t.read(path)
+	tpl, err := t.parse("template", path, options...)
 	if err != nil {
 		return nil, err
 	}
-
-	tpl := template.New("template")
-	for _, opt := range options {
-		tpl = opt(tpl)
+	buf := &bytes.Buffer{}
```

Comment:
maybe `bytes.NewBuffer(nil)` is a better way to initialize a buffer? Just a nit though.

Reply:
Done!